src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -80,6 +81,18 @@ inline int strncasecmp(const char *s1, const char *s2, size_t n) {
   return _strnicmp(s1,s2,n);
 }
 
+// VS doesn't provide strtok_r, which is a POSIX function.  Instead, it
+// provides the same function under the name strtok_s.  Note that this is
+// *not* the same as the C99 Annex K strtok_s.  VS provides that function
+// under the name strtok_s_l.  Make strtok_r a synonym so we can use that name
+// in shared code.
+const auto strtok_r = strtok_s;
```

Comment:
Just 1 question, is strtok\_r in this VC definition a function pointer? I don't think I've ever seen code do this before. Also, I think Annex K is C11, not C99